src/main/java/com/google/cloud/genomics/dataflow/pipelines/IdentityByState.java

```diff
@@ -89,7 +89,8 @@ public static void validateOptions(Options options) {
 
   }
 
-  private static final String VARIANT_FIELDS = "variants(start,calls(genotype,callSetName))";
+  private static final String VARIANT_FIELDS =
+      "variants(alternateBases,calls(callSetName,genotype),end,referenceBases,start)";
```

Comment:
I added a tip to the JavaDocs in the Streamer classes and also each place they are used in the example pipelines.